raiden_installer/installer_server.py

```diff
@@ -0,0 +1,220 @@
+    # Installer code running in a bundle
+    template_folder = static_assets_absolute_path('templates')
+    static_folder = static_assets_absolute_path('static')
+
+    app = Flask(
+        __name__,
+        template_folder=template_folder,
+        static_folder=static_folder
+    )
+else:
+    # Installer code running live
+    app = Flask(__name__)
+
+
+@app.route('/', methods=['GET', 'POST'])
+def install_raiden():
+    if request.method == 'POST':
+        keystore_pwd = request.form['keystore-pwd']
+        proj_id = request.form['proj-id']
+        network = 'goerli'
+
+        '''
+        Installer Step 1
+
+        Create keystore directory and
+        retrieve keyfile and address.
+        '''
+        keyfile = keystore.make_keystore(
+            KEYSTORE_DIR,
+            keystore.generate_keyfile_name(),
+            keystore_pwd
+        )
+
+        with open(keyfile, 'r') as f:
+            keyfile_content = json.load(f)
+
+        address = keyfile_content['address']
+
+        '''
+        Installer Step 2
+
+        Build the ETH RPC endpoint URL
+        '''
+        eth_rpc = raiden_config.eth_rpc_endpoint(proj_id, network)
+
+        '''
+        Installer Step 3
+
+        Grab ETH and confirm that the
+        account address has a balance.
+        '''
+        eth_acquisition = funding.goerli_funding(address)
+        w3 = Web3(HTTPProvider(eth_rpc))
+
+        while w3.eth.getBalance(to_checksum_address(address)) <= 0:
+            time.sleep(1)
+            print('Not enough ETH on account address')
+
+        '''
+        Installer Step 4
+
+        Acquire tokens for PFS and Monitoring services
+        and retrieve the user deposit contract address.
+        '''
+        private_key = keystore.get_private_key(keyfile_content, keystore_pwd)
+        pfs_monitoring_funding = funding.PfsAndMonitoringFunding(
+            w3,
+            to_checksum_address(address),
+            private_key
+        )
+        user_deposit_address = pfs_monitoring_funding.user_deposit_address
+
+        mint_tokens = pfs_monitoring_funding.mint_tokens(
+            TOKEN_AMOUNT,
+            GAS_REQUIRED_FOR_MINT,
+            GAS_PRICE
+        )
+        print('Status')
+        print(mint_tokens)
+
+        approve_deposit = pfs_monitoring_funding.approve_deposit(
+            TOKEN_AMOUNT,
+            GAS_REQUIRED_FOR_APPROVE,
+            GAS_PRICE
+        )
+        print('Status')
+        print(approve_deposit)
+
+        make_deposit = pfs_monitoring_funding.make_deposit(
+            TOKEN_AMOUNT,
+            GAS_REQUIRED_FOR_DEPOSIT,
+            GAS_PRICE
+        )
+        print('Status')
+        print(make_deposit)
+
+        '''
+        Installer Step 5
+
+        Create a plain txt pwd file
+        and generate a TOML config.
+        '''
+        plain_txt_pwd = raiden_config.PlainTxtPwd(CONFIG_DIR, keystore_pwd)
+        plain_txt_pwd.create_plain_txt_pwd_file()
+
+        global config_file
```

Comment:
I posted in the dev-questions channel on Rocket.Chat about this and decided for this approach as a way to pass data across requests.

Reply:
But you don't need to pass data across requests, do you? It seems you just want to be able to read/write to the same file. Unless I misunderstood the functionality, the only thing that needs to be known is the name of the file, so that you can open/close it freely.